schema-next.yaml

```diff
@@ -131,6 +131,24 @@ versions:
               status: system.processes.status
             apply_to_metrics:
               - system.processes.count
+        # https://github.com/open-telemetry/semantic-conventions/pull/330
+        - rename_attributes:
```

Comment:
I think it's better to read if the attributes listed here are split by their "namespace". Like I did for the system metrics https://github.com/open-telemetry/semantic-conventions/pull/89/files#diff-3ddd62d7281d98d63c3e92f5ae8cd7776971b44ca39f52b0140dc02363dfebda

Reply:
Resolved in [5ad3a95](https://github.com/open-telemetry/semantic-conventions/pull/330/commits/5ad3a95ab2ddaf681fef56fbe757d1dedda68744) (rebased so it moved to a different place)